### PR TITLE
Update zipkin-php to the latest version

### PR DESCRIPTION
**Pull request by Reasno**
Is it ok to upgrade zipkin dependency to the latest version? I need to access the version with https://github.com/openzipkin/zipkin-php/issues/76#issue-334247322 resolved.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "jcchavezs/zipkin-opentracing",
+    "name": "reasno/zipkin-opentracing",
```
Comment (jcchavezs) [inline composer.json]: I think you added this change wrongly. Maybe you want to revert?

File: composer.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
     "description": "A Zipkin bridge with OpenTracing",
     "type": "library",
     "require": {
-        "openzipkin/zipkin": "1.1.1",
+        "openzipkin/zipkin": "1.3.4",
```
Comment (jcchavezs) [inline composer.json]: Use `^1.3.4` and then I will merge.

**Comment by jcchavezs** [comment]
Thanks for this. I just added a couple of comments.

**Comment by Reasno** [comment]
fixed

**Review by jcchavezs — APPROVED** [review]
Thanks for this!

**Pull request merged**